File: JavaScript/countdown/countdown.js
/** 
 * Write a function called countdown that accepts a number as a parameter and every 1000 milliseconds 
 * decrements the value and console.logs it. Once the value is 0 it should log “DONE!” and stop.
 */

const countdown = (num) => {
    let x = num;
    let cd = setInterval(() => {
        if (x == 0) {
            console.log("Done!");
            clearInterval(cd);
        } else {
            console.log(x);
            x--;
        }
    }, 1000);
}

countdown(10)

File: Python/countdown/countdown.py
""" 
 Write a function called countdown that accepts a number as a parameter and every 1000 milliseconds 
 decrements the value and console.logs it. Once the value is 0 it should log “DONE!” and stop.
 """

from time import sleep

def countdown(num):
    x = num
    while x > 0:
        print(x)
        sleep(1)
        if x == 1:
            print("Done!")
        x -= 1

countdown(10)
